spec/order_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'order'
+
+describe Order do 
+  let(:order) { Order.new }
+  it 'initializes with an empty list of items' do 
+    expect(order.items).to be_empty
+  end 
+
+  it 'adds selections to an order' do 
+    updated_items = order.add_to_cart("Margherita")
+    expect(updated_items).to include("Margherita")
+  end 
+  
+  it 'calculates the total of an order' do
+    order.add_to_cart({ item: "Margherita", price: 10, quantity: 2 })
```

Comment:
I see that you're using a selection from menu to add to an order in your feature test. This could be shown here by mocking the `select` method on Menu and using that in your order class